app/business/services/EmailService.scala

```diff
@@ -0,0 +1,17 @@
+package business.services
+
+import common._
+import scala.concurrent.Future
+
+/**
+ * Created by liviu.ignat on 3/24/2015.
+ */
+trait IEmailService {
+  def sendCreatedAccountEmail(to: String): Future[LastError]
```

Comment:
will look into Either soon then :). You're right error management is probably not even needed for an email service